flake-compat.nix

```diff
@@ -1,6 +1,7 @@
 # The `default.nix` in flake-compat reads `flake.nix` and `flake.lock` from `src` and
 # returns an attribute set of the shape `{ defaultNix, shellNix }`
 
-import (fetchTarball https://github.com/edolstra/flake-compat/archive/master.tar.gz) {
-  src = ./.;
-}
+import (fetchTarball
```

Comment:
why this change? also do we need that file at all?

Reply:
I think it was just an auto-format.This would be for stable nix users, if they want to build the package. Might be worth supporting that use case until flakes become the default.

Reply:
I somewhat doubt any of this cli would work without flakes... but ok.

Reply:
Actually yeah, now that you mention that, it probably doesn't make sense :sweat_smile: